packages/teams-js/src/internal/communication.ts

```diff
@@ -66,7 +66,8 @@ export function initializeCommunication(validMessageOrigins: string[] | undefine
   Communication.currentWindow = Communication.currentWindow || ssrSafeWindow();
   Communication.parentWindow =
     Communication.currentWindow.parent !== Communication.currentWindow.self
-      ? Communication.currentWindow.parent
+      ? Communication.currentWindow
+          .top /* Obviously, don't want to call this Communication.parentWindow anymore. Also, frameless */
```

Comment:
Curious what you mean by "Also, frameless"?

Reply:
@erinha frameless mode doesn't use post messages to talk with the host, so we'd have to make sure what we're doing also works in those scenarios